src/geom/Transform.cc

```diff
@@ -105,16 +105,22 @@ typename FromEndpoint::Array Transform<FromEndpoint, ToEndpoint>::tranInverse(
 }
 
 template <typename FromEndpoint, typename ToEndpoint>
+Transform<ToEndpoint, FromEndpoint> Transform<FromEndpoint, ToEndpoint>::getInverse() const {
+    std::shared_ptr<ast::Mapping> inverse = _frameSet->getInverse();
```

Comment:
Hmm... is there a way to unit test for the information you say is currently missing?